tests/components/freedompro/test_sensor.py

```diff
@@ -0,0 +1,20 @@
+"""Tests for the Freedompro sensor."""
+from homeassistant.helpers import entity_registry as er
+
+
+async def test_sensor_get_state(hass, init_integration):
```

Comment:
Can the test be expanded to actually check the sensor state, as well as test all three sensors?

For example parametrize the test:
```suggestion
@pytest.mark.parametrize(
    "entity_id, device_class, state",
    [
        ("sensor.garden_humidity_sensor", "humidity", 0),
        ("sensor.temperatureSensor", "temperature", 0),
    ]
)
async def test_sensor_get_state(hass, init_integration):
    ...
```

Reply:
Note: This is not a complete suggestion, the test needs to be updated to use the new parameters

Reply:
sorry, my mistake